code/game/objects/items/storage/fancy.dm

```diff
@@ -23,10 +27,10 @@
 	var/spawn_type
 	/// How many of the things to fill this storage with.
 	var/spawn_count = 0
-	/// Whether the container is open or not
-	var/is_open = FALSE
-	/// What this container folds up into when it's empty.
-	var/obj/fold_result = /obj/item/stack/sheet/cardboard
+	/// Whether the container is open, always open, or closed
+	var/is_open = CONTAINER_CLOSED
+	/// What material do we get when we fold this box?
+	var/foldable = /obj/item/stack/sheet/cardboard
```

Comment:
should still be named `foldable_result` or something, since `foldable` alone implies it's a boolean to someone who isn't being careful